files/scripts/update-cluster.sh

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/env bash
+set -eu
+
+function patchCluster {
+  local configLocation=$1
+  local authToken=$2
+  echo "Patching cluster"
+  waitService "http://graphdb-cluster-proxy:7200/proxy/ready" "$authToken"
+  curl -o patchResponse.json -isSL -m 15 -X PATCH  --header "Authorization: Basic ${authToken}" --header 'Content-Type: application/json' --header 'Accept: application/json' -d @"$configLocation" 'http://graphdb-cluster-proxy:7200/rest/cluster/config'
+    if grep -q 'HTTP/1.1 200' "patchResponse.json"; then
+      echo "Patch successful"
+    elif grep -q 'Cluster does not exist.\|HTTP/1.1 412' "patchResponse.json" ; then
+      echo "Cluster does not exist"
```

Comment:
if a cluster does not exist, you'll return a success, is this ok?

Reply:
As this job will always execute last because of it's weight if by this point there is no cluster it's either expected or something has gone terribly wrong and you are probably aware of it, so yes I think it's okay.